tiledb/libtiledb/enum.cc

```diff
@@ -163,9 +163,8 @@ void init_enums(py::module& m) {
         .value("S3", TILEDB_S3)
         .value("AZURE", TILEDB_AZURE)
         .value("GCS", TILEDB_GCS)
-        .value("HDFS", TILEDB_HDFS);
```

Comment:
Removing this is a breaking change. Are we sure we want that? I didn't remove `TILEDB_HDFS` in my Core PR for this reason.

Reply:
I defer to you for any major decisions. I made this simple PR by naively removing all mentions of HDFS. I just added this line back. Please let me know if there are any other changes you would like me to make.

Reply:
@kounelisagis is going to add a warning here.

Reply:
Did we decide against adding a warning? I don't feel strongly one way or the other; but the file `enum.cc` is no longer edited by this PR.

Reply:
I couldn't find a way to do so.

Reply:
We had discussed this and my suggestion was to not do anything since this is an internal API.